package.json

```diff
@@ -1,19 +1,7 @@
 {
   "name": "@google-cloud/logging",
-  "description": "Stackdriver Logging Client Library for Node.js",
   "version": "7.3.0",
-  "license": "Apache-2.0",
-  "author": "Google Inc.",
-  "engines": {
-    "node": ">=8.10.0"
-  },
-  "repository": "googleapis/nodejs-logging",
-  "main": "./build/src/index.js",
-  "types": "./build/src/index.d.ts",
-  "files": [
-    "build/src",
-    "build/protos"
-  ],
+  "description": "Stackdriver Logging Client Library for Node.js",
```

Comment:
stable refactors from the generator would alleviate a lot of diff noise. consider sorting object keys.

see also `src/v2/config_service_v2_client_config.json`'s `non_idempotent` swap

Reply:
great suggestion! Will sort package json from microgenerator and pick up the changes here.